styles/components/button.css

```diff
@@ -0,0 +1,118 @@
+.btn {
+  cursor: pointer;
+  border-radius: var(--border-rds);
+  padding: 10px 20px;
+  border: none;
+  font-size: 1rem;
+  transition: 0.7s;
+  display: flex;
+  gap: 10px;
+  align-items: center;
+}
+
+.btn-primary {
+  background: var(--primary);
+  border: 2px solid var(--primary);
+  color: var(--white);
+}
+
+.btn-danger {
+  background: var(--danger);
+  border: 2px solid var(--danger);
+  color: var(--white);
+}
+
+.btn-info {
+  background: var(--info);
+  border: 2px solid var(--info);
+  color: var(--white);
+}
+
+.btn-warning {
+  background: var(--warning);
+  border: 2px solid var(--warning);
+  color: var(--white);
+}
+
+.btn-success {
+  background: var(--success);
+  border: 2px solid var(--success);
+  color: var(--white);
+}
+
+.btn-glass {
+  background: rgba(255, 255, 255, 0.3);
+  box-shadow: var(--shadow);
+  backdrop-filter: blur(6.5px);
+  -webkit-backdrop-filter: blur(6.5px);
+  color: var(--black);
```

Comment:
suggestion - variable naming can be better, also I thing I would suggest remove color property from here as you haven't given any color to the base class, so default it will take color as  black